### DISCUSSION TITLE
vectorr tap is giving this error command terminated with exit code 137

### DISCUSSION THREAD

**Discussion by vibhavachar** [vectordotdev/vector#21123] [General]
We upgraded the EKS cluster from 1.22 to 1.28 after that we get 137 code.  Its giving the same for 0.34, 0,38 version. 

kubectl exec -it vector-ha-deploy-1a-585c954f7-sgzv6 -n vector-events  -- vector help
command terminated with exit code 137

Any one faced this issue ? Will be very difficult to debug without access to vector tap.

**Comment by vibhavachar** [comment]
We are running vector production setup , any help of guidance is highly appreciated.

**Comment by jszwedko** [comment]
It looks like error code 137 indicates a lack of memory. See https://refine.dev/blog/kubernetes-exit-code-137/. I'd suggest trying to increase the memory limit.

**Comment by vibhavachar** [comment]
We have given 20gb ram , utilization is just 1 gb.

**Reply by jszwedko** [reply]
Interesting, I'm not sure why k8s would be killing the container due to OOM then. I'd suggest researching that error code: https://www.google.com/search?q=kubernetes+137&oq=kubernetes+137&gs_lcrp=EgZjaHJvbWUqBwgAEAAYgAQyBwgAEAAYgAQyBwgBEAAYgAQyBwgCEAAYgAQyBwgDEAAYgAQyCAgEEAAYFhgeMggIBRAAGBYYHjIKCAYQABgPGBYYHjIGCAcQRRg80gEIMjY5OWowajeoAgCwAgA&sourceid=chrome&ie=UTF-8
